### ISSUE TITLE
Issue with migrating reportportal database to another machine

### ISSUE THREAD

**Issue by elvovsky** [reportportal/reportportal#1969]
We have a current installation of RP v.5.7.3 running in docker on linux host running Ubuntu 18.04. That version of OS will be EOL in April, unless you subscribe for Ubuntu advantage. I tried to upgrade on a copy of this server and ran into some issues with specific packages (not related to RP).  So, I've built a new vm running Ubuntu 22.04. I 've followed install doc. All is good, I can get initial install working, however when I try to import backup from the db I run into many issues (errors with importing the PostgreSQL). I get a bunch of errors in the output, and it does not seem to import. So, in my mind, since I can get the default install with default login working, this is just DB import issue. I would like to make sure that I am following the right steps. The only two things that I saw about that are in the cheat sheet. That is how to backup and restore (commands) . Let me know what other information you would want from me and i can upload it.

**Comment by irynakozak2** [comment]
@hlebkanonik Gleb, could you please suggest here?

**Comment by hlebkanonik** [comment]
Hi @elvovsky,

Try to copy `./data/postgres/` and `./data/storage` to a new server in the same location and permissions. After that, start services with the command `docker compose -p reportportal up -d --force-recreate` from the directory where located `./data`

**Comment by elvovsky** [comment]
Ok, I will give it a try and let you know

**Comment by Timremmert** [comment]
I had the same issues after migrating from an old version to 5.7.4! Be sure that you use the same Postgres-Version like before and run the Migrations-Script in the database: https://github.com/reportportal/migrations

**Comment by elvovsky** [comment]
Thanks @Timremmert . In my case, there is no upgrade. It is just moving the data to the new set up with exact same versions. @hlebkanonik Hleb, the initial quick test seemed to be successful. I was able to login with the username and password that I had set up in the original . However, I will need to make sure that I can have developers run the test is and be able to post the data. That may take longer.
